storage/rocksdb/rdb_utils.cc

```diff
@@ -27,6 +27,17 @@
 /* MyRocks header files */
 #include "./ha_rocksdb.h"
 
+/*
+  Both innobase/include/ut0counter.h and rocksdb/port/port_posix.h define
+  CACHE_LINE_SIZE.
+*/
+#ifdef CACHE_LINE_SIZE
+#  undef CACHE_LINE_SIZE
+#endif
+
+/* RocksDB header files */
+#include "util/compression.h"
```

Comment:
These header files should be internal to the rocksdb submodule though and not meant for public consumption.